tests/api_workflow/test_api_workflow_datasets.py

```diff
@@ -29,6 +30,16 @@ def test_dataset_name_exists__own_not_existing(self):
             dataset_name="not_existing_dataset"
         )
 
+    def test_dataset_exists(self):
+        assert self.api_workflow_client.dataset_exists(dataset_id="dataset_1_id")
+        assert not self.api_workflow_client.dataset_exists(
+            dataset_id="non_existing_dataset_id"
+        )
+
+    def test_dataset_exists__raises_error(self):
+        with self.assertRaises(ApiException):
+            self.api_workflow_client.dataset_exists(dataset_id=None)
```

Comment:
Can you please also assert the status code of the raised exception? To make sure it is not 404.